Inc/GraphicsMemory.h

```diff
@@ -92,6 +92,9 @@ namespace DirectX
         
         explicit operator bool () const { return mSharedResource != nullptr; }
 
+        bool operator == (const SharedGraphicsResource& other) const { return mSharedResource.get() == other.mSharedResource.get(); }
```

Comment:
Note that comparison operators of ``std::shared_ptr`` have the same semantics...